colorDC1/Dockerfile

```diff
@@ -128,13 +126,18 @@ RUN echo "gea_user ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 # Switch to GEA user
 COPY ./jupyter_notebook_config.json /opt/conda/etc/jupyter/jupyter_notebook_config.json
 COPY ./jupyter_notebook_config.py /home/gea_user/.jupyter/jupyter_notebook_config.py
-COPY ./.bashrc /home/gea_user/.bashrc
-RUN chown -R gea_user:gea_user /home/gea_user/.jupyter
-RUN chmod -R 777 /home/jovyan
-RUN chown gea_user:gea_user /home/gea_user/.bashrc
+COPY ./_bashrc /home/gea_user/.bashrc
+COPY ./_bash_prompt /home/gea_user/.bash_prompt
+RUN mkdir /home/gea_user/work && \
+    chown -R gea_user:gea_user /home/gea_user
+WORKDIR /home/gea_user/work
 USER gea_user
 #
 #
+# The HOME environment variable is set in the parent image, so we have to override it in the Dockerfile.
+ENV HOME /home/gea_user
```

Comment:
This overrides the HOME environment variable setting from the parent Docker image.